book/SUMMARY.md

```diff
@@ -1,9 +1,17 @@
-[Introduction](./README.md)
+# Reth Book
 
-# Getting Started
+## Getting Started
+- [Introduction](./README.md)
+- [System requirements](./getting_started/system_requirement.md)
+- [Installation Reth](./getting_started/README.md)
 
-- [Installation](./installation.md)
-- [First Steps with Reth]()
-<!-- An overview of all the flags, how they work and how to configure the node   -->
-- [Configuring The Node]()
-- [Running Reth]()
+## [Fundamentals](./fundamentals/README.md)
+- [Introduction](./fundamentals/introduction.md)
```

Comment:
What's the intended difference between `Getting Started > Introduction` and `Fundamentals > Introduction`?

Reply:
I'll keep this one and remove it from Fundamentals.